manager/assets/modext/widgets/core/modx.searchbar.js

```diff
@@ -97,25 +110,35 @@ MODx.SearchBar = function(config) {
                     this.tpl.type = null;
                 }
             }
-            ,focus: this.focusBar
-            ,blur: this.blurBar
+            ,focus: {
+                fn: function(){
+                    this.focusBar();
+                }
+            }
+            ,blur: {
+                fn: function(){
+                    this.blurBar()
+                }
+            }
             ,scope: this
         }
+
     });
+
     MODx.SearchBar.superclass.constructor.call(this, config);
-    //this.setKeyMap();
+    this.setKeyMap();
```

Comment:
That's a good point. I think it might be hard to find a shortcut that does not conflict with anything, so maybe the best fix is to get rid of the "opening uberbar" shortcut? The primary reason I renabled that was for closing the uberbar with ESC.